packages/screens/RWA/components/EstateCard/EstateSubmit.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { TextInput, View } from "react-native";
+
+import { PrimaryButton } from "../../../../components/buttons/PrimaryButton";
+import { useTheme } from "../../../../hooks/useTheme";
+import { neutralA3 } from "../../../../utils/style/colors";
+
+export const EstateCardViewProperty: React.FC = () => {
+  const theme = useTheme();
+
+  return (
+    <PrimaryButton
+      color={theme.primaryButtonColor}
+      text="View Property"
+      width={284}
+      squaresBackgroundColor={theme.backgroundColor}
+      style={{ flex: 1 }}
+    />
+  );
+};
+
+export const EstateCardWailistInput: React.FC = () => {
+  const [value, onChangeValue] = React.useState<string>("");
+  const theme = useTheme();
+
+  return (
+    <View style={{ top: 3 }}>
+      <View style={{ flex: 1, flexDirection: "row" }}>
+        <TextInput
+          value={value}
+          onChangeText={onChangeValue}
+          style={{
+            width: 150,
+            flex: 1,
+            padding: 10,
+            color: theme.textColor,
+            borderWidth: 1,
+            borderColor: theme.borderColor,
+            borderRadius: 10,
+            fontSize: 12,
+          }}
+          placeholder="mail@teritori.com"
+          placeholderTextColor={neutralA3}
+        />
+        <PrimaryButton
+          color={theme.primaryButtonColor}
+          text="Join the Waitlist"
+          style={{ flex: 1, right: 18 }}
```

Comment:
`right: 18` makes the button overlap the input. The mockup is respected, but appears broken when the input is focused or the button pressed
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/839c37ac-80b8-4aad-ae38-25346c143518)

Maybe just reduce the input width and add a space bewteen the input and the button is ok, np. Instead of adding 
 a ton if props on existing components or making a new one. (And remove `right: 18`)
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/34780457-fc08-4e0d-a194-8a01057e12e8)

Reply:
Yep i agree with you for the compromise 👍